ui/page/components/wallet_sync_info.go

```diff
@@ -462,6 +462,8 @@ func (wsi *WalletSyncInfo) progressStatusDetails() (int, string) {
 func (wsi *WalletSyncInfo) layoutAutoSyncSection(gtx C) D {
 	return layout.Flex{}.Layout(gtx,
 		layout.Rigid(func(gtx C) D {
+			autoSync := wsi.wallet.ReadBoolConfigValueForKey(sharedW.AutoSyncConfigKey, false)
+			wsi.syncSwitch.SetChecked(autoSync)
```

Comment:
This appears to be the only place where users see a checkbox indicating that the wallet is syncing. This should use `wallet.IsSyncing() || wallet.IsSynced()` instead.

Reply:
Updated.

Users will have to use the sync btn next time they start the app.